core/src/main/java/org/elasticsearch/script/Script.java

```diff
@@ -161,14 +166,22 @@ public String getLang() {
         return params;
     }
 
+    public XContentType getContentType() {
```

Comment:
Needs explanation here too because it isn't obvious what this means to most scripts.